SQL/sqlRules.json

```diff
@@ -0,0 +1,42 @@
+{
+  "SQL_Server_Auth_EntraIDIntegrated": {
+    "expected": true,
+    "explanation": "The server should be integrated with Azure Active Directory for authentication and authorization. For more information: https://learn.microsoft.com/en-us/azure/azure-sql/database/authentication-aad-configure?view=azuresql&tabs=azure-powershell",
+    "function": "hasEntraIDIntegrationEnabled"
+  },
+  "SQL_Server_Security_Auditing" : {
+    "expected": true,
+    "explanation": "The server should have auditing enabled. For more information: https://learn.microsoft.com/en-us/azure/azure-sql/database/auditing-setup?view=azuresql",
+    "function": "hasAuditingEnabled"
+  },
+  "SQL_Server_Security_TLS1.2": {
+    "expected": true,
+    "explanation": "The server should have TLS 1.2 enabled. For more information: https://learn.microsoft.com/en-us/azure/azure-sql/database/connectivity-settings?view=azuresql&tabs=azure-portal#minimal-tls-version",
+    "function": "hasMinimumTLSVersion12"
+  },
+  "SQL_Server_Security_NetworkAccess": {
+    "expected": true,
+    "explanation": "The server should have public network access disabled. For more information: https://learn.microsoft.com/en-us/azure/azure-sql/database/connectivity-settings?view=azuresql&tabs=azure-portal ",
+    "function": "hasPublicNetworkAccessDisabled"
+  },
+  "SQL_Server_Security_OutboundNetworkAccess": {
+    "expected": true,
+    "explanation": "The server should have restrictions on outbound network access disabled. For more information: https://learn.microsoft.com/en-us/sql/sql-server/install/configure-the-windows-firewall-to-allow-sql-server-access?view=sql-server-ver16 ",
```

Comment:
Should be changed to https://learn.microsoft.com/en-us/azure/azure-sql/database/outbound-firewall-rule-overview